From: Alexander Graf <agraf@suse.de>
Subject: [PATCH] Fix highbank sata proc_name
Patch-Mainline: No, but reported upstream

Mkinitrd relies on proc_name to match the module name. Sata_highbank
failed to do so, so let's teach it to behave.

Signed-off-by: Alexander Graf <agraf@suse.de>

Index: current/drivers/ata/sata_highbank.c
===================================================================
--- current.orig/drivers/ata/sata_highbank.c
+++ current/drivers/ata/sata_highbank.c
@@ -251,7 +251,7 @@ static const struct ata_port_info ahci_h
 };
 
 static struct scsi_host_template ahci_highbank_platform_sht = {
-	AHCI_SHT("highbank-ahci"),
+	AHCI_SHT("sata_highbank"),
 };
 
 static const struct of_device_id ahci_of_match[] = {
